spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -213,8 +213,17 @@ def setup_toolbar(self):
                         context='variable_explorer',
                         name='search', parent=self)
 
+        self.refresh_button = create_toolbutton(
+            self, text=_("Refresh variables"),
+            icon=ima.icon('redo'),

```

Comment:
Please add a new entry for this in the `ima` module and use `mdi.refresh` for it.

Reply:
And please name the entry `refresh`.